### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by amejid**
## Awesome books: Milestone 1

In this milestone, the following were implemented:

- No linter errors
- Use the correct GitHub flow
- Create a collection that keeps a list of books
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Display all books saved in the collection in the top part of the page.
- Preserve data in the browser's memory by using localStorage

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/* eslint-disable no-use-before-define */
+function loadFromStorage() {
+  const books = JSON.parse(localStorage.getItem('booksInfo'));
+  displayBooks(books);
+}
+loadFromStorage();
+
+function removeBook() {
+  books.splice(this, 1);
```
Comment (FrederickMih) [inline main.js:23]: - [Optional] Let's practice making use of the Javascript ES6 filter() method since it doesn't mutate the original array.

**Review by FrederickMih — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (amejid) [reply main.js:23]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

**Comment by amejid** [comment]
Thank you for reviewing and approving our code.

**Pull request merged**